src/Altinn.Auth.AuditLog.Persistence/AuthorizationEventRepository.cs

```diff
@@ -57,7 +58,8 @@ INSERT INTO authz.eventlogv1(
                 @operation,
                 @ipaddress,
                 @contextrequestjson,
-                @decision
+                @decision,
+                @useridentifier
```

Comment:
I think we should follow the pattern for the rest. So subjectsystemuserid

Reply:
fixed